src/components/Avatar/Avatar.vue

```diff
@@ -51,7 +51,17 @@
 			:src="avatarUrlLoaded"
 			:srcset="avatarSrcSetLoaded"
 			alt="">
-		<div v-if="hasMenu" class="icon-more" />
+		<Popover
+			v-if="hasMenu"
+			placement="auto"
+			:open="contactsMenuOpenState">
+			<template>
```

Comment:
this template element seems redundant, the could should work without if I'm not mistaken

Reply:
removed now